storybookjs/storybook
# fix : Addon panel does not update after disabling/enabling an addon

Issue: https://github.com/storybookjs/storybook/issues/22158

## What I did

- Update `useList` in `tabs.hooks.tsx` to show new added tabs

## How to test

- Start a sandbox template: yarn task --task dev --template react-vite/default-ts
- Add actions: { disable: true } and see it update correctly
- Change to actions: { disable: false } or remove the line altogether, and see that the panel get updated

## Demo

https://user-images.githubusercontent.com/51731962/233590377-198c1b9b-37b3-453b-ac0b-0bcf193f28c6.mov

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.
